docker-multinode/common.sh

```diff
@@ -274,9 +269,9 @@ kube::multinode::restart_docker(){
       else
         DOCKER_CONF="/etc/default/docker"
         
-        kube::helpers::file_replace_line ${DOCKER_CONF} `# Replace content in this file` \
-          "--bip" `# Find a line with this content...` \
-          "OPTIONS=\"\$OPTIONS --mtu=${FLANNEL_MTU} --bip=${FLANNEL_SUBNET}\"" `# ...and replace the found line with this line`
+        kube::helpers::file_replace_line ${DOCKER_CONF} \ # Replace content in this file
+          "--bip" \ # Find a line with this content...
```

Comment:
This "--bip" won't work (https://github.com/kubernetes/kube-deploy/issues/154) because grep interprets --bip as an unknown option instead of as the thing to search for.